components/StackedList.tsx

```diff
@@ -0,0 +1,42 @@
+import clsx from 'clsx'
+import { DetailedHTMLProps, HTMLAttributes, ReactNode } from 'react'
+
+type BaseProps<T = HTMLDListElement> = DetailedHTMLProps<HTMLAttributes<T>, T>
+
+export interface StackedListProps extends BaseProps {
+  children: ReactNode
+}
+
+const StackedList = (props: StackedListProps) => {
```

Comment:
if we type this like so, I think we can avoid specifying children here

```suggestion
const StackedList: React.FC<BaseProps> = (props) => {
```

Reply:
I forgot one thing after I committed this suggestion. `React.FC` could be problematic both in type definitions and early assumptions. References:

- https://github.com/typescript-cheatsheets/react#section-2-getting-started
- https://fettblog.eu/typescript-react-why-i-dont-use-react-fc

Reverted in 5e74a0a8eddacc7fc7d40f85e4dc6e7cbffaab24.